extensions/wikia/Wall/WallThread.class.php

```diff
@@ -191,10 +191,16 @@ public function purgeLastMessage() {
 		WikiaDataAccess::cachePurge( $key );
 	}
 
+	/**
+	 * TODO: used by ForumController::boardThread() method only. Move it there?
```

Comment:
I want to avoid conflicts with big refactoring changes from @TK-999 that are in QA column.